src/app/models/stack.ts

```diff
@@ -0,0 +1,15 @@
+import { Card } from 'src/app/models/card';
+import { Learner } from 'src/app/models/learner';
+
+export class Stack{
+
+    constructor(
+        public stackId:number,
+        public name:string,
+        public description:string,
+        public creatorId:number,
+        public cards:Card[],
+        public learners:Learner[]
```

Comment:
Stack should not have the learners array. We will be grabbing stacks from learners instead of learners from stacks.